assets/css/admin/forms/_form-row.scss

```diff
@@ -0,0 +1,12 @@
+.edd-form-row {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 12px;
+
+	.is-column {
```

Comment:
Should this have an `edd-` prefix?

Reply:
This should be moved inside `.edd-form-group` so if can only be used with that class which is already prefixed.

Reply:
I'm mostly concerned about another plugin that's not prefixing having an `is-column` class and those styles affecting ours.

Reply:
@ashleyfae Good point. I don't like how the naming _looks_ but it's a safer route to take.

@robincornett I still believe it should be nested under `.edd-form-group`.

Reply:
I guess that's why I was wondering about using the `--` in the class--I've seen that recommended to represent variations ([ref](https://medium.com/@davehouse_80809/bem-for-everyone-else-89ccc8ad66f2)).

Totally fine with moving it, too--working on that now.

Reply:
@robincornett Yep, you were correct with the more verbose name the first time around.

I di think the variation name makes more sense being `.edd-form-group--is-column` as that is a variation of the current block where `.edd-form-group--row` makes it seem like its a variation of the parent block, which goes against BEM -- even though it's super loose BEM to begin with.

Reply:
@robincornett if we are going with verbose names to help avoid conflicts I'm swinging more towards using `.edd-form-row__column`. The naming aligns more with BEM vs. WordPress' loose BEM for variants and it also helps make it clearer that the parent wrapper is needed for the columns to work correctly.

```html
<div class="edd-form-row">
  <div class="edd-form-row__column" />
</div>
```

That leaves the contents of the columns completely arbitrary which leaves more flexibility if we find that we need something in addition to `.edd-form-group` in the future.